src/Plugin/EloquaAppCloudDecisionResponderInterface.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Drupal\eloqua_app_cloud\Plugin;
+
+use Drupal\Component\Plugin\PluginInspectionInterface;
+
+/**
+ * Defines an interface for Eloqua AppCloud Decision Responder plugins.
+ */
+interface EloquaAppCloudDecisionResponderInterface extends PluginInspectionInterface {
+
+  /**
+   * Method that gets executed when a Decision Service is invoked from within the
+   * Eloqua UI.
+   *
+   * @param object $record
+   * object jsondecoded from Eloqua transmission.
+   *
+   * @return null
```

Comment:
- [ ] Need more detail on this.  I assume this is actually boolean?  Do we support NULL as a neutral response?  Exceptions?  Critical, as this will be our only form of documentation for...  Awhile.